src/components/Header/NavBar/NavBar.jsx

```diff
@@ -0,0 +1,53 @@
+import styles from "../../../styles/navbar.module.css";
+import { Link } from "react-router-dom";
+import { HOME, LIBRARY, PLAYLIST, HISTORY } from "../../../routing/paths";
+import { Contact } from "../../index";
+import { usePathName } from "../../../hooks";
+
+export const NavBar = () => {
+  const { pathUrl } = usePathName();
+  return (
+    <div className={styles.navbar}>
+      <ul className={styles.nav_ul}>
+        <li>
+          <Link to={HOME} className={pathUrl === "/" ? styles.active : null}>
+            HOME
+          </Link>
+        </li>
+        <li>
+          <Link
```

Comment:
The library data isn't visible, make sure the user is able to see some data whether or not he is logged in. please fix it